contrib/kind.sh

```diff
@@ -1032,45 +1032,46 @@ install_ingress() {
   run_kubectl apply -f "${DIR}/ingress/service-nodeport.yaml"
 }
 
+METALLB_DIR="/tmp/metallb"
 install_metallb() {
```

Comment:
Note: inv dev-env already installs controller-gen, so we can drop the code that installs it:
https://github.com/metallb/metallb/blob/cb39f244ac299ad4ebf4be64ac81b0907a0a4e0c/tasks.py#L297